src/SmartComponents/RunTaskModal/InputParameters.js

```diff
@@ -34,31 +47,24 @@ export const InputParameter = ({ parameter, setDefinedParameters }) => {
   };
 
   return (
-    <Form className="pf-u-pb-lg">
+    <Form className="pf-v5-u-pb-lg">
       <FormGroup
         label={parameter.key}
         isRequired={parameter.required}
         type="text"
-        helperText={parameter.description}
-        helperTextInvalid={
-          parameter.required ? 'This parameter is required' : null
-        }
         fieldId="name"
-        validated={
-          parameter.required && paramText === '' ? ValidatedOptions.error : null
-        }
       >
         <TextInput
           value={paramText}
           type="text"
-          onChange={updateParams}
-          validated={
```

Comment:
On TextInput the `validated` should still be available, or?

Reply:
@bastilian `validated` is now a state and is passed to `HelperTextItem` below.